index.bs

```diff
@@ -1185,6 +1214,10 @@ a [=trigger state=] |triggerState|:
     :: false
     : [=attribution trigger/aggregation coordinator=]
     :: [=default aggregation coordinator=]
+    : [=attribution trigger/serialized private state tokens=]
+    :: «»
+    : [=attribution trigger/aggregatable source registration time configuration=]
+    :: "<code>[=aggregatable source registration time configuration/exclude=]</code>"
```

Comment:
To be clear, this value doesn't matter, right? The fake trigger is never used to create aggregatable reports.

Reply:
Yes, this is irrelevant, the fake trigger is only used to create fake event-level reports.